extensions/typescript-language-features/package.json

```diff
@@ -38,6 +38,11 @@
     "jsonc-parser": "^3.2.0",
     "semver": "5.5.1",
     "vscode-tas-client": "^0.1.63",
+    "vscode-uri": "^3.0.3",
+    "@vscode/sync-api-client": "^0.7.2",
+    "@vscode/sync-api-common": "^0.7.2",
+    "@vscode/sync-api-service": "^0.7.3",
+    "typescript": "latest",
```

Comment:
Do any changes in this PR rely on TS 5.0+?

The `typescript` dependency probably should be bumped under `extensions/package.json` instead but since we can't ship 5.0 beta to users in January, I can help look into a workaround

Reply:
No specific version, I just needed access to `tsserverlibrary` in webserver. If this is already available elsewhere, I can remove it.